ui/package.json

```diff
@@ -239,7 +235,6 @@
     "paths": [
       "lib/config-ui",
       "lib/core",
-      "lib/css",
```

Comment:
this addon is no longer necessary, per [this comment](https://github.com/hashicorp/vault/pull/26245/files#r1548744545), so i removed it